sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/DataSource.scala

```diff
@@ -440,7 +440,7 @@ case class DataSource(
     // ordering of data.logicalPlan (partition columns are all moved after data column).  This
     // will be adjusted within InsertIntoHadoopFsRelation.
     InsertIntoHadoopFsRelationCommand(
-      outputPath = outputPath,
+      outputFsPath = outputPath,
```

Comment:
Could you undo this redundant change, @LantaoJin ?

Reply:
This field overwrites the outputPath in DataWritingCommand and the return type is different (Path vs Option[Path]), so I rename this.